annotation/lib/http.dart

```diff
@@ -196,3 +199,14 @@ class FormUrlEncoded {
   final mime = 'application/x-www-form-urlencoded';
   const FormUrlEncoded();
 }
+
+/// Runs the given function with the passed argument in a new isolate and returns
```

Comment:
Please move this code block in a separated file and export it in `retrofit.dart` file